core/controllers/reader.py

```diff
@@ -566,6 +566,47 @@ class ExplorationStartEventHandler(base.BaseHandler):
     """Tracks a learner starting an exploration."""
 
     REQUIRE_PAYLOAD_CSRF_CHECK = False
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'params': {
+                'schema': {
+                    'type': 'dict',
+                    'properties': []
+                }
+            },
```

Comment:
Be more specific here, what are the possible params.

Reply:
@vojtechjelinek should I add extra validators for params? Regarding the keys in params, I think it is empty. I tried adding a schema under `'properties' : []` considering `'v'` as a key which led to an error and gave an empty list of extra keys and `'v'` in the list of missing keys.

Reply:
Why are the params even send from the frontend if they are empty? The frontend should probably be modified in that case.

Reply:
Oh I see, should I make this as a separate PR? I do not exactly know how to do this but I will work on it.

Reply:
Hey, @SD-13 recently looked into this and we came to conclusion that you solution is fine, because the `params` can have varying structure depending on the particular exploration.

Reply:
Oh I see, so can I let it remain as such? No changes needed here right

Reply:
Yes